test/gtest/ucp/test_ucp_am.cc

```diff
@@ -797,3 +840,33 @@ UCS_TEST_P(test_ucp_am_nbx_rndv_dts, rndv, "RNDV_THRESH=256")
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_am_nbx_rndv_dts);
+
+
+class test_ucp_am_nbx_rndv_memtype: public test_ucp_am_nbx_rndv {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant>& variants) {
+        add_variant_memtypes(variants,
+                             test_ucp_am_nbx_eager_memtype:: get_test_variants);
+    }
+
+    void init()
```

Comment:
`modify_config("RNDV_THRESH", 0)` here as for eager
so, no need to do it for each test case

Reply:
ok (btw i have just 1 test case)